### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16552,6 +16555,8 @@ optmized->optimized
 optoin->option
 optoins->options
 optomism->optimism
+opulate->populate, opiate,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest opulent too

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18527,10 +18534,13 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provident,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest provider?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18527,10 +18534,13 @@ provate->private, provide,
...
 provdie->provide
 provdied->provided
 provdies->provides
 provding->providing
+proveable->provable
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Remove this and put a note in the relevant issue, it's British English:
https://www.collinsdictionary.com/dictionary/english/proveable

**Comment by luzpaz** [comment]
ready for merge

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -18555,10 +18561,12 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provided, provident,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: You've typed `provided` twice, not `provider` the second time.

I was going to add an issue to catch this in the tests, but they already do:
https://travis-ci.org/github/codespell-project/codespell/jobs/668123853?utm_medium=notification&utm_source=github_status

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: should be fixed in c0b2141

**Comment by luzpaz** [comment]
> Can you fix the test and the outstanding comment.

looks like it's fixed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -18555,10 +18561,12 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provider, provident,
 provdie->provide
 provdied->provided
 provdies->provides
 provding->providing
+provedd->proved
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I wonder if this should suggest `provided`?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: aa79ac5

**Pull request merged**